tests/linear_solve_test.py

```diff
@@ -78,8 +78,10 @@ def test_solve_dense(self):
     x = linear_solve.solve_lu(matvec, b)
     x2 = jax.numpy.linalg.solve(A, b)
     x3 = linear_solve.solve_iterative_refinement(matvec, b)
+    x4 = linear_solve.solve_qr(matvec, b)
     self.assertArraysAllClose(x, x2)
     self.assertArraysAllClose(x, x3)
+    self.assertArraysAllClose(x, x4, atol=1e-4)
```

Comment:
The fact that atol=1e-4 is needed is strange (I agree that it indeed does not converge without it. We should report a bug to the JAX team.